models/model.py

```diff
@@ -2,6 +2,7 @@
 from tensorflow.keras.utils import plot_model
 import pickle
 import json
+from include import RUN_LOCAL
```

Comment:
import inutilisé

Reply:
yes, it is used in the function model_V4